src/styles/blocks/card.scss

```diff
@@ -0,0 +1,116 @@
+.card {
+  display: flex;
+  margin-right: 48px;
+  margin-bottom: 46px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Move it element selector. In you case it can be something like class="container__card card"